aslprep/cli/workflow.py

```diff
@@ -150,14 +150,14 @@ def build_boilerplate(config_file, workflow):
         from shutil import copyfile
         from subprocess import CalledProcessError, TimeoutExpired, check_call
 
-        from pkg_resources import resource_filename as pkgrf
+        from aslprep.data import load as load_data
 
         # Generate HTML file resolving citations
         cmd = [
             "pandoc",
             "-s",
             "--bibliography",
-            pkgrf("aslprep", "data/boilerplate.bib"),
+            load_data.readable("boilerplate.bib"),
```

Comment:
`readable` is just a `Path`-like that supports `read_bytes()` and `read_text()`. You probably just want:

```suggestion
            load_data("boilerplate.bib"),
```

Reply:
Oh nice! Thanks!